docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2538,6 +2682,12 @@ definitions:
             stream_id:
               type: string
               description: the id of the stream that consumes data from this partition
+            assign_type:
+              type: string
+              description: |
+                - `hard`: partition can't be transferred to another stream until the stream is closed;
+                - `soft`: partition can be transferred to another stream in a case of rabalance or if another stream
```

Comment:
in case of rebalance, or if (3 changes)

Reply:
sorry, didn't get it

Reply:
in `a` case of r`a`balance or if-> in case of r`e`balance`,` or if